### DISCUSSION TITLE
uup_download_macos.sh can't be opened by Rosetta on m1 MacBook Pro

### DISCUSSION THREAD

**Discussion by lsz16828** [utmapp/UTM#3508] [UTM Help (macOS)]
**Describe the issue**
While I was trying to install win10 virtual machine on my m1 chip MacBook Pro, I followed the guide and downloaded the win10 uup dump for win10, then I tried to run uup_download_macos.sh but it couldn't be opened by Rosetta. On  right clicking the uup_download_macos.sh file and then "show information", there was only one open method, which was "text editor"

**Configuration**
* UTM Version: 
Latest utm
* OS Version: 
Mac OS Monterey 12.1
* Intel or Apple Silicon? 
Apple Silicon

Any kindly advice/tip is appreciated, I'm a new bird for Mac, and I'll inherit your spirit and help other new hands in future, my mail address is 583614678@qq.com, thanks.

**Comment by osy** [comment]
https://apple.stackexchange.com/questions/428768/on-apple-m1-with-rosetta-how-to-open-entire-terminal-iterm-in-x86-64-architec

**Comment by adespoton** [comment]
Would it make sense to rename "uup_download_macos.sh" as "uup_download_macos.command" to avoid this issue?  That way, Terminal.app will be the default owner of the file and the OS won't try to open it as a binary using Rosetta.
